com.unity.render-pipelines.universal/Editor/ShaderGraph/Targets/UniversalLitSubTarget.cs

```diff
@@ -650,26 +697,39 @@ static class LitDefines
             public static readonly KeywordDescriptor ClearCoat = new KeywordDescriptor()
             {
                 displayName = "Clear Coat",
-                referenceName = "_CLEARCOAT 1",
+                referenceName = "_CLEARCOAT",
                 type = KeywordType.Boolean,
                 definition = KeywordDefinition.ShaderFeature,
                 scope = KeywordScope.Local,
             };
 
-            public static readonly DefineCollection ComplexLit = new DefineCollection()
+            public static readonly KeywordDescriptor SpecularSetup = new KeywordDescriptor()
             {
-                {CoreKeywordDescriptors.UseFragmentFog, 1},
-                {ClearCoat, 1},
+                displayName = "Specular Setup",
+                referenceName = "_SPECULAR_SETUP",
+                type = KeywordType.Boolean,
+                definition = KeywordDefinition.ShaderFeature,
+                scope = KeywordScope.Local,
+                stages = KeywordShaderStage.Fragment
             };
         }
         #endregion
 
         #region Keywords
         static class LitKeywords
         {
+            public static readonly KeywordDescriptor ReceiveShadowsOff = new KeywordDescriptor()
+            {
+                displayName = "Receive Shadows Off",
+                referenceName = ShaderKeywordStrings._RECEIVE_SHADOWS_OFF,
+                type = KeywordType.Boolean,
+                definition = KeywordDefinition.ShaderFeature,
+                scope = KeywordScope.Local,
+            };
+
             public static readonly KeywordDescriptor GBufferNormalsOct = new KeywordDescriptor()
             {
-                displayName = "GBuffer normal octaedron encoding",
+                displayName = "GBuffer normal octahedron encoding",
```

Comment:
Should this actually be "octahedral"?

Reply:
Yes, as the adjective form of octahedron?   :)